zubhub_backend/zubhub/activitylog/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Why do we have an empty test file here? Are we going to add tests in the future?

Reply:
this file was generated when I ran migrations. The same file existed in other folders too so I didn't remove it.